wezterm-gui/src/overlay/copy.rs

```diff
@@ -161,9 +162,13 @@ impl CopyOverlay {
         render.dirty_results.add(search_row);
         render.update_search();
 
+        let shared_render: Arc<Mutex<CopyRenderable>> = Arc::new(Mutex::new(render));
```

Comment:
unless it is required by the compiler, I generally prefer to avoid using explicit type annotations, as it results in less code to read, which makes it easier, for me at least, to grok more code at a glance:

```suggestion
        let shared_render = Arc::new(Mutex::new(render));
```